shell/client/styles/_topbar.scss

```diff
@@ -75,8 +75,7 @@ body>.topbar {
   background-color: $topbar-background-color;
   color: #ccc;
 
-  // Using 0 rather than CSS default ("auto") allows introjs to highlight elements in the topbar.
-  z-index: 0;
+  z-index: 1;
```

Comment:
I haven't read this thread it complete detail, but note that the `z-index: 0` was added by Asheesh yesterday. Setting `z-index: 0` _also_ creates a new stacking context, which is something I was concerned about when merging that change. Are the problem you describe present for `z-index: 0` (as checked into master), or only for `z-index: 1`?